x-pack/plugins/security_solution/public/detections/components/callouts/read_only_alerts_callout/index.tsx

```diff
@@ -14,16 +14,16 @@ const readOnlyAccessToAlertsMessage: CallOutMessage = {
   type: 'primary',
   id: 'read-only-access-to-alerts',
   title: i18n.READ_ONLY_ALERTS_CALLOUT_TITLE,
-  description: <p>{i18n.READ_ONLY_ALERTS_CALLOUT_MSG}</p>,
+  description: i18n.readOnlyAlertsCallOutBody(),
 };
 
 const ReadOnlyAlertsCallOutComponent = () => {
-  const [{ hasIndexWrite }] = useUserData();
+  const [{ hasIndexUpdateDelete }] = useUserData();
 
   return (
     <CallOutSwitcher
       namespace="detections"
-      condition={hasIndexWrite != null && !hasIndexWrite}
+      condition={hasIndexUpdateDelete != null && !hasIndexUpdateDelete}
```

Comment:
Nit: I think this should suffice and cover boolean, undefined or null scenarios.

```suggestion
      condition={!hasIndexUpdateDelete}
```

Reply:
Nope 🙂  `hasIndexUpdateDelete != null` means index privileges have been loaded. We cannot show this callout until then. Nulls causing confusion, as always! Might make sense to refactor at some point.

Reply:
Ah I see, I see! Thanks!